src/detection/Bootprint.cc

```diff
@@ -30,7 +30,9 @@ namespace lsst { namespace afw { namespace detection {
 Bootprint::Bootprint(std::shared_ptr<geom::SpanSet> inputSpans,
           geom::Box2I const & region): lsst::daf::base::Citizen(typeid(this)),
                                        _spans(inputSpans),
-                                       _region(region) {}
+                                       _region(region) {
+                                           setPeakSchema(PeakTable::makeMinimalSchema());
+                                       }
```

Comment:
I don't know. This was just me (and my editors suggestion) though. I was attempting to have all the initializations line up since they were one unit of thought. and then the body is indented as much so that the closing braces will match the indention level of the line containing the opening.